packages/polyfills-loader/package.json

```
@@ -1,7 +1,8 @@
 {
   "name": "@open-wc/polyfills-loader",
-  "version": "0.3.1",
+  "version": "0.3.2",
```

no need to bump version - CI takes care of it

```suggestion
  "version": "0.3.1",
```